modules/ROOT/pages/migrate-from-an-earlier-version-of-bonita-bpm.adoc

```diff
@@ -224,138 +223,130 @@ xref:back-up-bonita-bpm-platform.adoc[Back up your platform] and database in cas
+29. The update is now complete.
 
-The migration is now complete. If you were using a custom Look & Feel before migration, test it on the new version before applying it to your migrated platform.
+== Special cases
 
-== Migration of processes with 6.x forms and case overview pages
+=== Migrating processes with 6.x forms and case overview pages
 
-Until the version 7.0.0, Bonita used UI artifacts based on the Google Web Toolkit (GWT) technology: process instantiation, task execution forms and case overview page.
+Until Bonita 7.0.0, Bonita used UI artifacts based on the Google Web Toolkit (GWT) technology: process instantiation forms, task execution forms and case overview page.
 The runtime support for those forms and pages was removed in 7.8.0.
 
-It means that if one or more processes on the migrated server uses 6.x forms or overview page, the migration to a version above 7.7.x cannot be performed directly. The following lines explain how to migrate to a version 7.8.0.
-
-Specifically if you are migrating from a 6.x version:
-
-* Migrate to the 7.0.0 using the migration tool 1.x.
-* Migrate to the last 7.7.x version, using the migration tool 2.x.
-* Redesign your process to use contracts at process instantiation and task execution levels, and recreate all your forms and case overview pages in the Studio using the UI Designer or your favorite IDE, so that they use  xref:contracts-and-contexts.adoc[contracts]. For more information, go to  xref:migrate-a-form-from-6-x.adoc[migrate a form from 6.x]
-* Upload the new version of all your processes using contracts, new forms, and new case overview pages.
-* Make sure the versions of the processes using 6.x forms have no more running instances, and disable them.
-* Perform the migration to the desired version.
+It means that if one or more processes on the target server uses 6.x forms or overview page, the migration to a Bonita 7.7.x and greater cannot be performed directly. The following lines explain how to migrate a process to Bonita 7.8.0, for example.
 
-If you are migrating from a 7.x version:
+* If you are updating from a Bonita 6.x version, follow this procedure:
+ . Update to Bonita 7.0.0 using the Migration Tool 1.x.
+ . Update to the last 7.7.x version, using the Migration Tool 2.x.
+ . Redesign your processes to use contracts at process instantiation and task execution levels, and recreate all your forms and case overview pages in Bonita Studio using the UI Designer or your favorite IDE, so that they use xref:contracts-and-contexts.adoc[contracts]. For more information, go to  xref:migrate-a-form-from-6-x.adoc[migrate a form from 6.x]
+ . Upload the new version of all your processes using contracts, new forms, and new case overview pages.
+ . Make sure the versions of the processes using 6.x forms have no more running instances, and disable them.
+ . Perform the updating proceudre to the desired version.
```

Comment:
Perform the updating proceudre => Perform the updating procedure

Reply:
thx